examples/python/Earthfile

```diff
@@ -1,12 +1,35 @@
-VERSION --global-cache 0.7
+# Common Python UDCs and Builders.
+VERSION --global-cache --use-function-keyword 0.7
 
-builder:
-    FROM ./../../earthly/python+python-base
+# Assuming the detailed setup from the second snippet is defined in relevant Earthly targets.
 
-    COPY --dir ./src .
+builder:
     DO ./../../earthly/python+BUILDER
 
-test:
+    # Copy the src directory and project-specific files into /poetry in the container.
+    # Adjusted to match your setup where the WORKDIR is set to /poetry.
+    COPY --dir src ./src
+    COPY pyproject.toml poetry.lock ./
+
+    # Install dependencies without creating a virtual environment inside the container.
+    RUN poetry install --no-root
+
+    # Lint the application using ruff.
+    RUN ruff ./src
+
+check:
     FROM +builder
 
-    RUN poetry run pytest
+    DO ./../../earthly/python+CHECK
+
+# format all Python files in the current project.  Local developers tool.
+#   CI target : false
+format:
+    LOCALLY
+
+    DO ./../../earthly/python+FORMAT --src=$(echo ${PWD})
+
+pythonlinter:
```

Comment:
target name should be spereated with `-` like `python-linter`

Reply:
yup I saw it in the CI error last night was planning to fix it this afternoon

Reply:
fixed in commit 98485de21c7132e2bdb4c0c02ba4797d54783e5f